lib/cb/models/implementations/job.rb

```diff
@@ -10,7 +10,7 @@ class Job < ApiResponseModel
                     :location, :distance, :latitude, :longitude, :location_formatted, :location_metro_city,
                     :description, :requirements, :employment_type,
                     :details_url, :service_url, :similar_jobs_url, :apply_url,
-                    :begin_date, :end_date, :posted_date,
+                    :begin_date, :end_date, :posted_date, :posted_time,
```

Comment:
I don't know what our limitations are to know if that'd hurt us with data format. I'd prefer it be a DateTime as well if we can.